### ISSUE TITLE
Improve loading time of large albums

### ISSUE THREAD

**Issue by tomudding** [GEWIS/gewisweb#1384]
Large albums (300+ photos) take a long time to load due to the amount of queries performed. We should try to find a way to optimize the queries or to dynamically load more photos while scrolling through the album (but then we also have to make sure that it works while having the photo viewer open).

_Originally reported by Bouke._

**Comment by tomudding** [comment]
"Infinite" scroll where we load ~50 photos at once might be a good enough solution. We'd have to add something into the URL to ensure we load as many items again when reloading the page (or via a direct visit).

However, I do not think PhotoSwipe v5 currently supports dynamically adding additional slides.

**Comment by tomudding** [comment]
It appears that checking the votes is incredibly expensive. It looks like it is loading an instance of `Member` for every single check. I tried to change the check to not load the entities (replacing `findOneBy` with `count`), however, that does not improve the situation. Also changing to `fetch: "EXTRA_LAZY"` does not help.

I also tried a native query, but some for unknown reason Doctrine keeps loading the related entities.
